packages/mattermost-redux/src/actions/channels.ts

```diff
@@ -516,7 +496,7 @@ export function fetchMyChannelsAndMembersREST(teamId: string): ActionFunc {
             dispatch(loadRolesIfNeeded(roles));
         }
 
-        return {data: {channels, members: channelMembers}};
+        return {data: {channels, channelMembers}};
```

Comment:
@M-ZubairAhmed is this key change not affecting any place we are using this function?

Reply:
i have checked the usage of this function, we are not using this actions return in any place, the reason for this change was to make it same as that of its gQL counterpart fetchChannelAndMembers